README.md

```diff
@@ -30,13 +30,15 @@ For Rails 3+ apps:
 
 ```ruby
 # In config/application.rb
+require "rack/attack"
```

Comment:
I think normally you don't need this require line in rails apps, given that by convention there's an auto-generated line that calls `Bundler.require` for all gems listed in the `Gemfile`.